packages/resolver/lib/sources/abi.ts

```diff
@@ -17,19 +17,30 @@ export class ABI {
     return null;
   }
 
-  async resolve(importPath: string, importedFrom: string = "") {
+  async resolve(
+    importPath: string,
+    importedFrom: string = "",
+    options: { compiler?: { name: string; version: string } } = {}
+  ) {
     let filePath: string | undefined;
     let body: string | undefined;
 
     if (!importPath.endsWith(".json")) {
       return { filePath, body };
     }
 
-    const resolution = await this.wrappedSource.resolve(importPath, importedFrom);
+    const resolution = await this.wrappedSource.resolve(
+      importPath,
+      importedFrom,
+      options
+    );
     if (resolution.body === undefined) {
       return { filePath, body };
     }
 
+    const { compiler } = options;
+    const solidityVersion = determineSolidityVersion(compiler);
```

Comment:
OK, this is actually a pre-existing thing rather than a change you made, but I think it's worth adding a comment in here somewhere that if we're in this code we can assume we're compiling Solidity (because this resolver source is turned off for Vyper).

Reply:
Sure; agreed!

Reply:
Added this as a doc comment